lib/inferno/dsl/fhir_evaluator/ig.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'rubygems/package'
+require 'zlib'
+
+module FhirEvaluator
```

Comment:
I wonder where this will be migrated, do you think it will be part of Inferno utils?
Does Inferno have similar util already (extracting key contents from IGs) that can reuse to extend?

Reply:
Good question. There's nothing in core that parses an IG that I'm aware of, but there are things in various test kits that this was inspired by, for example these in us-core-test-kit
https://github.com/inferno-framework/us-core-test-kit/blob/main/lib/us_core_test_kit/generator/ig_loader.rb
https://github.com/inferno-framework/us-core-test-kit/blob/main/lib/us_core_test_kit/generator/ig_resources.rb

Reply:
I believe https://oncprojectracking.healthit.gov/support/browse/FI-3440 concerns this issue, which is next phase of the integration. I am okay to do it in this PR though it looks like phase 1 is already overwhelmed by many issues.

Reply:
Yeah that's what I meant before - since we have a separate task for migrating this IG class and the class isn't used yet, it might be easier to just leave it out for now